executor/join.go

```diff
@@ -313,8 +319,88 @@ func (e *HashJoinExec) handleJoinWorkerPanic(r interface{}) {
 	e.joinWorkerWaitGroup.Done()
 }
 
+// Handling unmatched rows from the hash table
+func (e *HashJoinExec) handleUnmatchedRowsFromHashTableInMemory(workerID uint) {
```

Comment:
Can we merge the `inMemory` and `inDisk` functions?

Reply:
No. Because the in-memory hash table are processed in parallel, but another in sequence.

Reply:
No, because in-memory hash table is processed in parallel, but another in sequence.

Reply:
Can we define a function `handleUnmatchedRowsFromHashTable(workerID uint, inMemory bool)`?

Reply:
PTAL @fzhedu

Reply:
alread merged.